src/App.tsx

```diff
@@ -1,19 +1,161 @@
 import React from 'react';
-import './App.css';
+import './App.scss';
 
-import users from './api/users';
+import { TodoList } from './components/TodoList';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
 
-const App: React.FC = () => {
-  return (
-    <div className="App">
-      <h1>Add todo form</h1>
+const prepearedTodos = todosFromServer.map(todo => {
+  const users = usersFromServer.find((user) => user.id === todo.userId) || null;
```

Comment:
```suggestion
  const user = usersFromServer.find((user) => user.id === todo.userId) || null;
```
find возвращает одного пользователя, так что это user, а не users